app/controllers/addresses_controller.rb

```diff
@@ -1,21 +1,47 @@
 # frozen_string_literal: true
 
-# rubocop:disable Metrics/MethodLength
```

Comment:
Honestly, I feel that 5 lines per method is kind of insane. Our ChangeRequestsController definitely has some methods that are probably too long, but I really don't think breaking up these methods in AddressesController is helping with readability at all. I'd say feel free to leave this rubocop disable line in, and move the private methods back into the main methods.

We should probably separately configure rubocop to have a more reasonable maximum method length.